README.md

```diff
@@ -29,10 +29,18 @@ Before diving into the CLI, we recommend taking a look at the Rosetta API Docs:
 * [Construction API](https://www.rosetta-api.org/docs/construction_api_introduction.html)
 
 ## Install
+To install `rosetta-cli` from source, run:
 ```
 go get github.com/coinbase/rosetta-cli
 ```
 
+To download a binary directly for MacOS, run:
```

Comment:
Do we want to list installs for each OS? I was thinking do:
```
rosetta-cli-{YOUR REQUIRED VERSION HERE}-{YOUR OS HERE}
```

And then include a link to releases to identify

And then say, for example on mac with version 0.4.1:
```
curl -L https://github.com/coinbase/rosetta-cli/releases/download/v0.4.1/rosetta-cli-0.4.1-darwin-10.6-amd64 -o rosetta-cli; chmod +x rosetta-cli;
```